themes/Backend/ExtJs/backend/order/controller/batch.js

```diff
@@ -501,6 +513,15 @@ Ext.define('Shopware.apps.Order.controller.Batch', {
         // Add the resulting record to our result store
         resultStore.add(operation.resultSet.records);
 
+        var resultRecord = operation.resultSet.records[0];
+        if (resultRecord.get('success') === false) {
```

Comment:
Is the `=== false` really necessary? `!resultRecord.get('success')` should be just fine too!